metrics/metrics.go

```diff
@@ -45,6 +45,12 @@ func (m *LabelMap) Get(key string) *expvar.Int {
 	return m.Map.Get(key).(*expvar.Int)
 }
 
+// GetIncrFunc returns a function that increments the expvar.Int named by key.
+func (m *LabelMap) GetIncrFunc(key string) func(delta int64) {
```

Comment:
This is so it can implement the same interface as DomainVar which increments two parallel metrics.